examples/deno-router/README.md

```diff
@@ -0,0 +1,5 @@
+```

```

Comment:
remove examples/deno-router dir if this pr is given the ok, and turn it into docs